modeltrans/conf.py

```diff
@@ -77,6 +74,14 @@ def check_fallback_chain():
                 raise ImproperlyConfigured(message_fmt.format(l))
 
 
+def check_lang_code_duplicates():
+    available_languages = get_available_languages()
+    if len(available_languages) != len(set(available_languages)):
+        raise ImproperlyConfigured(
+            "Languages should not contains duplicates: {}".format(available_languages)
+        )
```

Comment:
This message should mention the setting a user should adjust in order to fix it (something like `MODELTRANS_AVAILABLE_LANGUAGES or LANGUAGES should not contain duplicates`)

Reply:
Changed the message.

Reply:
@jieter can you looks at this PR please?